ref/src/sw_core.F90

```diff
@@ -219,39 +190,12 @@ subroutine c_sw(sw_corner, se_corner, nw_corner, ne_corner,       &
           fx2(i, j) = fx1(i, j) * fx2(i, j)
         enddo
       enddo
-    endif
 
     ! Ydir:
     if (grid_type < 3 .and. .not. nested) then
       call fill2_4corners(delp, pt, 2, sw_corner, se_corner, ne_corner, nw_corner)
     end if
-    if ( hydrostatic ) then
-      do j = js-1, jep1+1
-        do i = is-1, iep1
-          if ( vt(i, j) > 0. ) then
-            fy1(i, j) = delp(i, j-1)
-            fy(i, j) = pt(i, j-1)
-          else
-            fy1(i, j) = delp(i, j)
-            fy(i, j) = pt(i, j)
-          endif
-          fy1(i, j) = vt(i, j) * fy1(i, j)
-          fy(i, j) = fy1(i, j) * fy(i, j)
-        enddo
-      enddo
-      do j = js-1, jep1
-        do i = is-1, iep1
-          delpc(i, j) = delp(i, j) + (fx1(i, j) - fx1(i+1, j) + fy1(i, j) - &
-                        fy1(i, j+1)) * rarea(i,j)
-#ifdef SW_DYNAMICS
-          ptc(i, j) = pt(i, j)
-#else
-          ptc(i, j) = (pt(i, j) * delp(i, j) + (fx(i, j) - fx(i+1, j) + &
-                       fy(i, j) - fy(i, j+1)) * rarea(i, j)) / delpc(i, j)
-#endif
-        enddo
-      enddo
-    else
+
       if (grid_type < 3) then
```

Comment:
Same indentation request here.  Please readjust the indentation of the former `else` branch to match the new structure of the code.

Reply:
Fixed indentation.